definitions/variable/tag_electricity.yaml

```
@@ -17,6 +17,12 @@
         description: Coal gases
     - Gas|Hydrogen:
         description: Hydrogen
+    - Gas|Synthetic Gas:
+        description: Synthetically produced gas
+    - Gas|Synthetic Gas|Green Methane:
+        description: Synthetically produced methane from renewable sources
+    - Gas|Synthetic Gas|Other Methane:
+        description: Synthtically produced methane from non-renewable sources
     - Gas|Biogas:
         description: Gas from biogenic sources
```

Suggest to add for consistency with the final-energy description of biogas
```suggestion
        description: Gas from biogenic sources (lower energy content than methane)
```